src/componets/Dragons.js

```diff
@@ -10,7 +10,6 @@ function Dragon() {
   }, [dispatch]);
   const changeReservation = (e) => {
     dispatch(changeDragonReservation(e.target.id));
-    console.log(e.target.id);
   };
```

Comment:
Please kindly remove the `console.log` from your code. It's normally used for debugging and using it in the code is dangerous because not all browsers support console.log as it is not part of the standard and is an extension of the DOM thus you should not count on its presence